pkg/osquery/tables/wifi_networks/internal/assets/get-networks.ps1

```diff
@@ -0,0 +1,34 @@
+function Get-Networks {
+    function Convert-ByteArrayToString {
+        [CmdletBinding()] Param (
+            [Parameter(Mandatory = $True, ValueFromPipeline = $True)] [System.Byte[]] $ByteArray
+            )
+
+        $Encoding  = New-Object System.Text.ASCIIEncoding
+        $Encoding.GetString($ByteArray)
+    }
+
+    Add-Type -Path ".\nativewificode.cs"
+    $WlanClient = New-Object NativeWifi.WlanClient
+
+    $WlanClient.Interfaces | ForEach-Object { $_.Scan() }
+
+    #check scan progress for each interface every 50 milliseconds
+    $scanInProgress = "false"
+    do {
+      $scanInProgress = "false"
+      $WlanClient.Interfaces | ForEach-Object {
+        $ip = $_.scanInProgress
+        if ($ip -eq "True") {
+          $scanInProgress = "true"
+        }
+      };
+      Start-Sleep -Milliseconds 100
```

Comment:
Can't decide if 100 is still tight. 250 maybe? Any discussion online?

Reply:
I can do 250 - MS says that the max time you should wait is 4 seconds, all drivers/interfaces/wifi modules/thingies should respond to a request for networks scans within 4 second as part of the qualification process for a 'made for windows' sticker. (I'm not 💯 on what the sticker thing actually means, but 4s seems like a reasonable bar). 

Empirically, 2s usually seemed like enough time when I was testing. but I don't have a good sense for how that translates to choosing a sleep time, other than I'd like to finish the waiting period as soon as possible.